AB_Hands_On/AB_Hands_On_FinalSection.ipynb

```diff
@@ -0,0 +1,1572 @@
+{
```

Comment:
<p>I feel like this section can be merged with the last section, infdata cheatsheet. Its not actually important that people know what infdata is at this point.</p><p> </p><p>In addition arviz has already be been used in the previous section for plots, I feel as though it should therefore be formally introduced there.</p><p> </p><p>My vote is to remove this section and move its recap points to the respective sections that are absorbing it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>